qml/MonitorItem.qml

```diff
@@ -307,10 +307,22 @@ Component
                     }
                 }
 
+                Row {
+                    spacing: UM.Theme.getSize("default_margin").height
+
+                    UM.Label {
+                        height: drivePrefixTextInput.height
+                        verticalAlignment: Text.AlignVCenter
+                        wrapMode: Text.WordWrap
+                        text: catalog.i18nc("@label", "If Print/Delete buttons are not working, change the Drive prefix above. \nMarlin firmware wants 'None', stock FlyingBear Ghost4/5 firmware wants '1:/'")
```

Comment:
I think for users of most printers, this message will carry little meaning. Let's change it a little bit.
```suggestion
                        text: catalog.i18nc("@label", "If the Print/Delete buttons aren't working, adjust the Drive prefix above. \nChoose 'None' or '1:/' based on your printer's firmware (e.g., 'None' for Marlin, '1:/' for FlyingBear Ghost).")
```

Reply:
ok will change this when I update the PR after I fixed the issue with the buttons below